Esterni/ManualeUtente/components/c4-Istruzioni_per_utilizzo.tex

```diff
@@ -0,0 +1,50 @@
+\section{Istruzioni per l'utilizzo}
+\subsection{Autenticazione}
+All'avvio verrà mostrata una schermata che servirà ad autenticarsi, nella quale bisognerà inserire nome utente e password del proprio account \gloman{Zextras Drive}. In caso di successo dell'operazione si avvierà il software, altrimenti verrà visualizzato un messaggio di errore e chiesto di reinserire le credenziali.
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale = 0.50]{components/img/login.png}
+    \caption{Vista del login}
+    \label{fig:Vista del login}
+\end{figure}
+\subsection{Selezione del percorso da sincronizzare}
+\label{sec:selezionepath}
+Al primo avvio dell'applicazione verrà richiesto di scegliere una cartella, nella quale verranno sincronizzati tutti i file. Non si deve scegliere la stessa cartella nella quale è contenuto il programma.
```

Comment:
cambierei
'una cartella, nella quale verranno sincronizzati tutti i file'
in
'la cartella i cui file verranno sincronizzati'

Reply:
Così però andresti a sottointendere che solo i file della cartella vengono sincronizzati, invece ci sono anche quelli in remoto